cypress/integration/user_profile/UploadUserProfileImage.feature

```diff
@@ -15,4 +15,4 @@ Feature: Upload UserProfile Image
   Scenario: Unable to change another user's avatar
     Given I am logged in with a "user" role
     And I visit another user's profile page
-    Then I cannot upload a picture
\ No newline at end of file
+    Then I cannot upload a profile picture
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-18T08:41:21Z" title="Thursday, June 18th 2020, 10:41:21 am +02:00">Jun 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,4 +15,4 @@ Feature: Upload UserProfile Image
   Scenario: Unable to change another user's avatar
     Given I am logged in with a "user" role
     And I visit another user's profile page
-    Then I cannot upload a picture
\ No newline at end of file
+    Then I cannot upload a profile picture
```


Cool you’ve clarified this as well! 😎👍🏼